platform-hub-web/src/app/shared/hub-api/hub-api.service.js

```diff
@@ -13,16 +15,23 @@ export const hubApiService = function ($rootScope, $http, $q, logger, events, ap
   service.revokeAdmin = revokeAdmin;
   service.getProjects = buildCollectionFetcher('projects');
   service.getProject = buildResourceFetcher('projects');
-  service.createProject = createProject;
-  service.updateProject = updateProject;
-  service.deleteProject = deleteProject;
+  service.createProject = buildResourceCreator('projects');
+  service.updateProject = buildResourceUpdater('projects');
+  service.deleteProject = buildResourceDeletor('projects');
   service.getProjectMemberships = getProjectMemberships;
   service.addProjectMembership = addProjectMembership;
   service.removeProjectMembership = removeProjectMembership;
   service.projectSetRole = projectSetRole;
   service.projectUnsetRole = projectUnsetRole;
   service.userOnboardGitHub = userOnboardGitHub;
   service.userOffboardGitHub = userOffboardGitHub;
+  service.getSupportRequestTemplates = buildCollectionFetcher('support_request_templates');
+  service.getSupportRequestTemplate = buildResourceFetcher('support_request_templates');
+  service.createSupportRequestTemplate = buildResourceCreator('support_request_templates');
+  service.updateSupportRequestTemplate = buildResourceUpdater('support_request_templates');
+  service.deleteSupportRequestTemplate = buildResourceDeletor('support_request_templates');
```

Comment:
Heheh – I always struggle with these terms. From an API perspective, `DELETE` is the HTTP verb.  So I figured we'd reuse that term instead.